### PR TITLE
Hooch now heals assistants.

### PR DESCRIPTION
**Pull request by naltee**
:cl: Ikacid
add: Hooch now heals assistants.
/:cl:

[why]: At the moment, hooch is only ever used as a stepping stone to Bacchus's Blessing, and considering it's kind of a pain in the ass to make to begin with, it's never served as a drink in its own right. Plus, greytide is a feature, not a bug, and I like the idea of the little shits working together to assemble the ingredients for it.

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -379,6 +379,11 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_name = "Hooch"
 	glass_desc = "You've really hit rock bottom now... your liver packed its bags and left last night."
 
+/datum/reagent/consumable/ethanol/hooch/on_mob_life(mob/living/carbon/M)
+	if(ishuman(M) && M.job == "Assistant")
```
Comment (Iamgoofball) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: you want assigned_role on the mind

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 2]
```diff
@@ -379,6 +379,11 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_name = "Hooch"
 	glass_desc = "You've really hit rock bottom now... your liver packed its bags and left last night."
 
+/datum/reagent/consumable/ethanol/hooch/on_mob_life(mob/living/carbon/M)
+	if(M.mind && M.mind.assigned_role in "Assistant")
```
Comment (SpaceManiac) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: `assigned_role` is a string, `"Assistant"` is a string, string-in-string won't return useful results

**Review by SpaceManiac — DISMISSED** [review]

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm (hunk 2 above)
Comment (Ikacid) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: I don't know why I did that.

**Comment by plapatin** [comment]
underground assistant speakeasies where greytides gather in swarms to chug hooch. i'd love to see it

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**